src/Panels/ChatLogPanel.cs

```diff
@@ -44,52 +47,46 @@ public ChatLogPanel()
         {
             _chatCommands = new List<ChatCommand>
             {
-                new ChatCommand("help", "Displays information about which commands can be displayed.", (command) =>
+                new ChatCommand("help", Translation.PullTranslation("ChatCMDhelp"), (command) =>
                 {
-                    foreach (var c in _chatCommands)
+                    foreach (ChatCommand c in _chatCommands)
                     {
                         PrintGameMessage($"/{c.Command} : {c.Description}");
                     };
                 }),
-                new ChatCommand("version", "Displays version information about the mod and game.", (command) =>
+                new ChatCommand("version", Translation.PullTranslation("ChatCMDversion"), (command) =>
                 {
                     PrintGameMessage("Mod Version  : " + Assembly.GetAssembly(typeof(CSM)).GetName().Version.ToString());
                     PrintGameMessage("Game Version : " + BuildConfig.applicationVersion);
                 }),
-                new ChatCommand("support", "Display support links for the mod.", (command) =>
+                new ChatCommand("support", Translation.PullTranslation("ChatCMDsupport"), (command) =>
                 {
                     PrintGameMessage("GitHub : https://github.com/DominicMaas/Tango");
                     PrintGameMessage("Discord : https://www.patreon.com/CSM_MultiplayerMod");
                     PrintGameMessage("Steam Workshop : https://steamcommunity.com/sharedfiles/filedetails/?id=1558438291");
                 }),
-                new ChatCommand("players", "Displays a list of players connected to the server", (command) =>
+                new ChatCommand("players", Translation.PullTranslation("ChatCMDplayers"), (command) =>
                 {
                     if (MultiplayerManager.Instance.CurrentRole == MultiplayerRole.None)
                     {
-                        PrintGameMessage("You are not hosting or connected to any servers.");
+                        PrintGameMessage(Translation.PullTranslation("ChatCMDnonplayers"));
                         return;
                     }
 
-                    foreach (var player in MultiplayerManager.Instance.PlayerList)
+                    foreach (string player in MultiplayerManager.Instance.PlayerList)
                     {
                         PrintGameMessage(player);
                     }
                 }),
-                new ChatCommand("hide-chat", "Hides the chat console. Can be displayed again by pressing the '`' key.", (command) =>
+                new ChatCommand("hide-chat", Translation.PullTranslation("ChatCMDhide-chat"), (command) =>
                 {
                     isVisible = !isVisible;
                 }),
-                new ChatCommand("donate", "Find out how to support the mod developers", (command) =>
```

Comment:
Why did you remove the `donate` command?

Reply:
IMO I don't think anyone would use the game command to donate to a mod. Since the donation is already pushed through the steam workshop page and the github. If someone actually used it. I believe it would make sense to have it in.